src/ngx_http_lua_balancer.c

```diff
@@ -15,10 +15,32 @@
 #include "ngx_http_lua_util.h"
 #include "ngx_http_lua_directive.h"
 
+typedef struct {
+    ngx_queue_t                    queue;
+    ngx_queue_t                    hnode;
+    ngx_uint_t                     hash;
+    ngx_http_lua_srv_conf_t       *lscf;
+    ngx_connection_t              *connection;
+    socklen_t                      socklen;
+    ngx_sockaddr_t                 sockaddr;
+    ngx_str_t                      host;
+    u_char                         hostbuf[32];
```

Comment:
better use a meaning full macro for the hardcode `32`.

And maybe host_data may be a better name? it's not actually a buffer.

Reply:
And better add a comment for this field.
```
try to avoid allocating memory from the connection pool
```